Project-1/README.md

```diff
@@ -159,13 +159,6 @@ $T\equiv (M\sqcap ∃parentOf^-.(\le3 ∃parentOf.M \sqcap \ge3 ∃parentOf.M))
 
 5. Select two formulas defined in ALC from question 4 to form the basis of a T-Box. Supplement this T-box with whatever other axioms you like, as well as an A-box, so that you ultimately construct a knowledge base K = (T,A). Provide a _model_ of K. This may be graphical or symbolic or both. 
 
-K = Knowledge base 
-K = (T, A)
-T = general statement (e.g. ∃rC [relation without a name; C = concept]
-A = specific statement (e.g. a:c; (a, b): r [relation with a name])
-
-If we want to describe a domain, we have general statenents to assert a general relation and we have specific statements to assert specific relations.
-
 (a)  B that says that x is a brother of y
 
 Adding:
```

Comment:
Question 2. Since (b) is a tautology, then its CNF is not "A ∨ ~C", but "C ∨ ~C" or "A ∨ ~A"

Q3. You say "O(x) = ∃z(P(zx) ∧ ∀y(P(uy) → x=y))" but here you should change from u to z.

Also, "T(x) = ∃w∃y∃z(P(wx) ∧ P(wy) ∧ P(wz) ∧ ~(x=y) ∧ ~(x=z) ∧ ~(y=z)) ∧ ∀u(P(wu) → u=x ∨ u=y ∨ u=z)) ", but you wanna point out y, z and u are male. So you miss some parts.

Q5. Don't introduce "HasParent" and "HasChild" because according to the question prompt, they are actually ParentOf and ParentOf-. Also, each formula of Q4 is an equivalence formula, so you should pick out two of them and then put them into your TBox.

Note! You should give a model for your K=(T,A), so that means that you should define a domain like U={a,b,c,d}, and then assign a subset of U to each concept and so on. But I don't see you did this in your answer.

Q6. ∃x∀y = "Something is identical with everything" or "There exists an x such that for every y." --- Please delete "Something is identical with everything" because we cannot read this from the quantifier. The same problem is for your interpretation of ∀x∃y

Q7. Congrats to your use of Mermaid! But your graph is wrong because it cannot differ between those two formula. To revise, you can delete one loop and save only one loop. 

Q9.

Your proof of (a) is begging the question because 

assume ¬(¬Px ∨ ¬Qx).
        Px ∧ Qx.

This step has already used DeMorgan's Law.

Q10. Natural deduction can use different derivation rules while tree proof can only use indirect proof

Reply:
Dear Karl, 

Thank you for your review. I really appreciate the time you took to take a look at my answers and give me feedback!

Q2: Ahh, yes. Thank you for catching this. 

Q3: I totally didn't realize I made this mistake. Thanks for catching it!

Ahh, I forgot about negating f(x) to show that y, z, and u are male. Does my edit for (e) T(x) look right?

T(x) = ∃w∃y∃z(P(w,x) ∧ P(w,y) ∧ P(w,z) ∧ ~(x=y) ∧ ~(x=z) ∧ ~(y=z) ∧ ~F(x) ∧ ~F(y) ∧ ~F(z) ∧ ∀u(P(w,u) → u=x ∨ u=y ∨ u=z) ∧ ~F(u))  

Q5: The reason why I introduced "HasParent" and "HasChild" is because I took from the Q5 prompt that we could add any other axioms that we wanted. Can "HasParent" and "HasChild" function as a new axiom and concept? Or am I misunderstanding it?

Ahh that's right. I will attempt to write out and define the domain. 

Q6: For ∃x∀y = "Something is identical with everything" and ∀x∃y = "Everything is identical with something", I got this from my symbolic logic course with Braun. I guess I will bracket these interpretations as a possible paraphrase. 

Q7: Got it. Thanks!

Q9: Hmm what do you suggest in replacement of it?

Q10: Wonderful. Thank you! 

This is really helpful! Many thanks!

Best, 
Jaron

Reply:
@JaronJCheung 

Q3(e), you should move "~F(u)" from the consequent to the antecedent. So it looks like: T(x) = ∃w∃y∃z(P(w,x) ∧ P(w,y) ∧ P(w,z) ∧ ~(x=y) ∧ ~(x=z) ∧ ~(y=z) ∧ ~F(x) ∧ ~F(y) ∧ ~F(z) ∧ ∀u(P(w,u) ∧ **~F(u)** → u=x ∨ u=y ∨ u=z) )

Reply:
@peihongx, sounds good. All these make sense. Thank you!

Best, 
Jaron